public/locales/fr/index.yml

```diff
@@ -0,0 +1,26 @@
+iosHint: Sur iOS, veuillez utiliser le navigateur Safari.
+errorClose: Fermez
+selectCertificate: Sélectionner le certificat
+selectCertificateDescription: |
+  Veuillez scanner le QR Code de votre certificat ou sélectionner une capture d'écran ou une page PDF avec le QR Code. 
+  Notez que la sélection d'un fichier directement à partir de la caméra n'est pas prise en charge.
+stopCamera : Arrêter la caméra
+startCamera : Démarrer la caméra
+openFile : Sélectionner un fichier (PDF, PNG)
+foundQrCode : Code QR trouvé !
+pickColor : Choisissez une couleur
+pickColorDescription : Choisissez une couleur de fond pour votre passe.
+colorWhite : blanc
+colorBlack : noir
+colorGrey : gris
+colorGreen : vert
+colorIndigo : indigo
+colorBlue : bleu
+colorPurple : violet
+colorTeal: bleu canard
+addToWallet: Ajouter à Wallet
```

Comment:
in french the app is called "Cartes" (cards literally), should we pick up the straight name or leave Wallet?

Reply:
Yes, we can use Cartes because most people will use that.

Reply:
@Nyalab But is that consistent with the `common:subtitle` text? Would you also refer to all wallet apps as "applications Cartes"?

Reply:
you're absolutely right

Reply:
no we should not because we are talking about "wallet applications" in the subtitle, that is something commonly known and right here we are specifically talking about the "Cartes" one. So we should leave it like this imho.